api/import.go

```diff
@@ -46,6 +46,7 @@ type ImportHandler struct {
 //
 // AddRoutes adds routes.
 func (h ImportHandler) AddRoutes(e *gin.Engine) {
+	e.Use(Required("imports"), Transaction)
```

Comment:
You'll need to create a RouteGroup before applying the middleware, otherwise it will be applied to the entire router.

```go
routeGroup := e.Group("/")
routeGroup.Use(Required("imports"), Transaction)
routeGroup.Get(SummariesRoot, h.ListSummaries)
...
```

Reply:
Thanks Sam! Updated.

Reply:
Good catch, Sam!